Source/Scene/Instanced3DModel3DTileContent.js

```diff
@@ -257,6 +249,8 @@ define([
         //>>includeEnd('debug');
         byteOffset += sizeOfUint32;
 
+        var gltfByteLength = byteLength - 32 - featureTableJSONByteLength - featureTableBinaryByteLength - batchTableJSONByteLength - batchTableBinaryByteLength;
```

Comment:
If you haven't already it's worth taking a look at how `Batched3DModel3DTileContent` computes this, it's a bit simpler.